SUMMARY.md

```diff
@@ -203,14 +203,16 @@
   * [How to Run a Hackathon](contributors/contributors/how-to-run-a-hackathon.md)
   * [Mattermost GitHub Sponsorship](contributors/contributors/github-sponsorship.md)
   * [Hacktoberfest Event Organizer Guide for Mattermost](contributors/contributors/hacktoberfest-mattermost-organizer-guide.md)
+* [MatterCon](contributors/mattercon/README.md)
```

Comment:
Why MatterCon above "Join Us"?

Reply:
I thought it might be a good segue into "Join us" after someone has watched the videos of the previous MatterCons. :)